find_tabular_errors/requirements.txt

```diff
@@ -0,0 +1,5 @@
+cleanlab==2.2
```

Comment:
```suggestion
```

Reply:
@cmauck10 don't need cleanlab from requirements for all examples, we always assume correct version is already installed